### DISCUSSION TITLE
autotuning failed to run: pdsh@GCRSANDBOX324: localhost: ssh exited with exit code 255

### DISCUSSION THREAD

**Discussion by dunalduck0** [deepspeedai/DeepSpeed#1792] [General]
I didn't install pdsh and DeepSpeed worked fine for model training.

I tried to the new feature Autotuning. It seems to require pdsh. Unfortunately, it still didn't work after installing pdsh as I've got the error:

**pdsh@GCRSANDBOX324: localhost: ssh exited with exit code 255**

What was the reason that pdsh is not required for model training but autotuning?
Why does pdsh need to "ssh localhost" for autotuning?

**Comment by GGGGGGXY** [comment]
same issue
I guess deep speed need pdsh to launch serval tasks to find the best config.
but model training , you can start the task by yourself

**Comment by SeekPoint** [comment]
+1
